stable/jenkins/values.yaml

```diff
@@ -268,6 +268,13 @@ master:
     enabled: false
     defaultConfig: false
     configScripts: {}
+    # Ignored if securityRealm is defined in master.JCasC.configScripts
+    securityRealm: |-
+      legacy
+    # Ignored if authorizationStrategy is defined in master.JCasC.configScripts
+    authorizationStrategy: |-
+      loggedInUsersCanDoAnything:
+        allowAnonymousRead: false
```

Comment:
These need to be multi-line strings to allow the default values to be overridden.